dao/src/main/scala/za/co/absa/enceladus/dao/EnceladusRestDAO.scala

```diff
@@ -45,7 +60,49 @@ object EnceladusRestDAO extends EnceladusDAO {
     .configure(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS, false)
     .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
 
+  private[dao] def getUserFromKeytab(keytabLocation: String): String = {
+    val keytab = KeyTab.getInstance(keytabLocation)
+    keytab.getOneName.getName
+  }
+    
+  def spnegoLogin(keytabLocation: String): Boolean = {
+    import scala.collection.JavaConversions._
+    
+    val principal = getUserFromKeytab(keytabLocation)
+    _userName = principal
+    
+    val template = new KerberosRestTemplate(keytabLocation, principal)
+    val url = s"$restBase/user/info"
+
+    log.info(s"Calling REST with spnego auth $principal $keytabLocation")
```

Comment:
```suggestion
    log.info(s"Calling REST with SPNEGO auth $principal $keytabLocation")
```

apparently, it should be written all in caps

Reply:
Changing.... though you know what I'm thinking ;)